config.py

```diff
@@ -352,3 +352,7 @@ def address_from_private_key(private_key):
     GE_DB_PORT = ''
     GE_DB_PASSWORD = ''
     GE_HTTP_PROVIDER = ''
+
+TRANSFERS_CREATED_SINCE = common_secrets_parser['REWARDS'].getint('created_since')
```

Comment:
Why secret?

Is `created_since` a quasi constant value?

Reply:
In the meantime yes.